File: lib/enums/group_tips_group_info_type.dart
/// 群资料变更消息类型
enum GroupTipsGroupInfoType {
  /// 非法值
  Invalid,

  /// 修改群名称
  ModifyName,

  /// 修改群简介
  ModifyIntroduction,

  /// 修改群公告
  ModifyNotification,

  /// 修改群头像URL
  ModifyFaceUrl,

  /// 修改群主
  ModifyOwner,

  /// 修改群自定义字段
  ModifyCustom,
}

File: ios/Classes/enums/GroupTipsGroupInfoType.swift
import  ImSDK;

//  Created by 蒋具宏 on 2020/2/15.
//
public enum GroupTipsGroupInfoType : Int{
    case Invalid = 0
    case ModifyName = 1
    case ModifyIntroduction = 2
    case ModifyNotification = 4
    case ModifyFaceUrl = 8
    case ModifyOwner = 16
    case ModifyCustom = 32
    
    /**
     * 根据腾讯云获得枚举
     */
    static func getByTIM_GROUP_INFO_CHANGE_TYPE(type : TIM_GROUP_INFO_CHANGE_TYPE)->GroupTipsGroupInfoType?{
        return (GroupTipsGroupInfoType(rawValue: type.rawValue) ?? nil);
    }
}
